examples/cppgc-object.rs

```diff
@@ -57,24 +47,31 @@ fn main() {
          mut rv: v8::ReturnValue| {
           let id = args.get(0).to_rust_string_lossy(scope);
 
-          let templ = v8::ObjectTemplate::new(scope);
-          templ.set_internal_field_count(2);
-
-          let obj = templ.new_instance(scope).unwrap();
-
-          let member = v8::cppgc::make_garbage_collected(
-            scope.get_cpp_heap().unwrap(),
-            Box::new(Wrappable {
-              trace_count: Cell::new(0),
-              id,
-            }),
-          );
-
-          obj.set_aligned_pointer_in_internal_field(
-            0,
-            &DEFAULT_CPP_GC_EMBEDDER_ID as *const u16 as _,
-          );
-          obj.set_aligned_pointer_in_internal_field(1, member.handle as _);
+          fn empty(
+            _scope: &mut v8::HandleScope,
+            _args: v8::FunctionCallbackArguments,
+            _rv: v8::ReturnValue,
+          ) {
+          }
+          let templ = v8::FunctionTemplate::new(scope, empty);
+          let func = templ.get_function(scope).unwrap();
+          let obj = func.new_instance(scope, &[]).unwrap();
```

Comment:
I think i've asked this somewhere and got an answer but forgot about it lol: Why do we need to create a FunctionTemplate instead of just?

```
let templ = v8::ObjectTemplate::new(scope);
```

Reply:
in order for the object to have a JSObjectWithEmbedderSlots it needs to be created with either FunctionTemplate or with ObjectTemplate with non-zero amount of embedder slots. just felt cleaner to use FunctionTemplate than to make random unused slots.